app/controllers/comments_controller.rb

```diff
@@ -1,12 +1,12 @@
 class CommentsController < ApplicationController
   skip_before_action :verify_authenticity_token
-  load_and_authorize_resource
   def new
```

Comment:
User is not able to comment on a post. Please check that a user is able to comment on a post.

Reply:
Fixed